packages/next/src/build/output/log.ts

```diff
@@ -31,12 +31,21 @@ function prefixedLog(prefixType: keyof typeof prefixes, ...message: any[]) {
   if (message.length === 0) {
     console[consoleMethod]('')
   } else {
-    console[consoleMethod](' ' + prefix, ...message)
+    // Ensure if there's ascii color escape it's concatenated into one string.
```

Comment:
```suggestion
    // Ensure if there's ANSI escape codes it's concatenated into one string.
```

ANSI color codes contain ASCII escape sequences.